partcad-cli/src/partcad_cli/click/command.py

```
@@ -328,13 +329,30 @@ def cli(ctx, verbose, quiet, no_ansi, package, format, **kwargs):
 
     if ctx.invoked_subcommand in commands_with_context:
         from partcad.globals import init
+        import partcad.user_config as pc_usr_config
 
         try:
-            ctx.obj = init(package)
+            user_config = UserConfig()
+            user_config.force_update = ctx.invoked_subcommand in commands_with_forced_update
+
+            for env_var, attrib in user_config_options:
+                value = kwargs.get(attrib, None)
```

Ruff (0.8.2)

Use `kwargs.get(attrib)` instead of `kwargs.get(attrib, None)`

Replace `kwargs.get(attrib, None)` with `kwargs.get(attrib)`

(SIM910)